client/components/Overview/components/Styles.jsx

```diff
@@ -14,18 +14,18 @@ const Styles = ({
   const getAllStyles = () => {
     axios.get('http://localhost:3000/api/getallstyles', { params: { id } })
       .then((results) => {
-        //  create an array with all styles (not in rows of 4)
-        setImageArray(results.data);
-        //  set the number of rows for thumbnail on main image
-        let numberOfRows = Math.ceil(results.data.length / 7);
-        //  split the thumbnails into n arrays of length 7
-        setThumbSplitArr([...Array(numberOfRows)]
-          .map((row, idx) => results.data
-            .slice(idx * 7, idx * 7 + 7)));
+        // //  create an array with all styles (not in rows of 4)
+        // setImageArray(results.data);
+        // //  set the number of rows for thumbnail on main image
+        // let numberOfRows = Math.ceil(results.data.length / 7);
+        // //  split the thumbnails into n arrays of length 7
+        // setThumbSplitArr([...Array(numberOfRows)]
+        //   .map((row, idx) => results.data
+        //     .slice(idx * 7, idx * 7 + 7)));
```

Comment:
Are these comments kept around in case of having to revert to a previously functioning system?

Reply:
Yep, it is primarily there in case I need to backtrack while not reverting commits.